introduction.md

```diff
@@ -1,12 +1,4 @@
-Hello
+My name is bbooth,
 
-blah
-akdfsfjl;kdajfklasdjflk
-asdf
-asdfasdfasdf
-asdfasdfasdfasdf
-asdfasdfasdfasdfasdf
-asdfasdfasdf
-
-
-JavaScript
+Skills
+* JavaScript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?